### PR TITLE
feat(opentelemetry-collector): add some MacOS-specific troubleshooting instructions

### PR DESCRIPTION
**Pull request by astencel-sumo**
## PLEASE READ

Following a recent back-end update, all contributors with a local clone or fork of our repository are required to run `yarn install`. This does not apply to [direct page edits](https://help.sumologic.com/docs/contributing/edit-doc/#minor-edits).

- [x] Yes, I've run `yarn install`
- [ ] No, does not apply to me

## Purpose of this pull request

This pull request adds some MacOS-specific commands to troubleshoot Sumo OpenTelemetry Collector distro on a Mac.

This is related to this [Slack thread](https://sumologic.slack.com/archives/C01D3SJQTK3/p1701199984898319?thread_ts=1692695576.628219&cid=C01D3SJQTK3), where Vijit had issues installing on MacOS and we realized we don't have any specific troubleshooting steps to help here.

If this is accepted, we should add similar instructions for Linux and Windows.

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions, .clabot
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/send-data/opentelemetry-collector/install-collector/macos.md
Comment (sumo-drosiek) [inline docs/send-data/opentelemetry-collector/install-collector/macos.md:342]: What if not? How to troubleshoot that?

Comment (astencel-sumo) [reply docs/send-data/opentelemetry-collector/install-collector/macos.md:342]: Good question, you can add a follow-up PR. 😅 This PR does not aim to be complete, it's an incremental improvement. I believe this addition is an improvement, but certainly not a complete troubleshooting doc.

**Comment by swiatekm-sumo** [comment]
These troubleshooting steps look like something the install script itself should do as well on MacOS, as it already does on Linux.

**Comment by kimsauce** [comment]
> @kimsauce, I was about approve this PR, but first I want your opinion. Do you think the layout of the Troubleshooting section is fine as it is, or should it be changed?

Let me check! thanks for the heads up.

File: docs/send-data/opentelemetry-collector/install-collector/macos.md
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/install-collector/macos.md:269]:
```suggestion
Here are some basic troubleshooting steps on macOS.
```

File: docs/send-data/opentelemetry-collector/install-collector/macos.md
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/install-collector/macos.md:342]:
```suggestion
The output should include `active count = 1` and `state = running`. This means the daemon is running.
```

**Comment by kimsauce** [comment]
> These troubleshooting steps look like something the install script itself should do as well on MacOS, as it already does on Linux.

@swiatekm-sumo @sumo-drosiek can you advise on this, whether further changes are needed? I'll keep it on hold until you approve.

**Review by kimsauce — CHANGES_REQUESTED** [review]
Minor edits

**Comment by kimsauce** [comment]
> These troubleshooting steps look like something the install script itself should do as well on MacOS, as it already does on Linux.

@swiatekm-sumo per @sumo-drosiek : "I've created gh issue to implement the change in installation script, but until it is done, I would go with documentation, so imho we should merge it."

**Review by kimsauce — APPROVED** [review]

**Pull request merged**
